src/main/java/org/primefaces/component/timeline/DefaultTimelineUpdater.java

```diff
@@ -101,21 +101,39 @@ public void clear() {
 
     @Override
     public PhaseId getPhaseId() {
-        return PhaseId.RENDER_RESPONSE;
+        return PhaseId.ANY_PHASE;
     }
 
     @Override
     public void beforePhase(PhaseEvent event) {
+        if (event.getPhaseId().equals(PhaseId.APPLY_REQUEST_VALUES)) {
```

Comment:
I would flip these around like `PhaseId.APPLY_REQUEST_VALUES.equals(event.getPhaseId())` so the constant is always on the left.

Reply:
OK, good observation. 👍